content/docs/quickstart/cpp.md

```diff
@@ -9,63 +9,111 @@ description: This guide gets you started with gRPC in C++ with a simple working
 
 #### gRPC
 
-To install gRPC on your system, follow the [instructions to install gRPC C++ via make](https://github.com/grpc/grpc/blob/master/src/cpp/README.md#make).
+Follow the gRPC installation instructions for your OS:
 
-To run the example code, please ensure `pkg-config` is installed on your
-machine before you build and install gRPC in the previous step, since the
-example `Makefile`s try to look up the installed gRPC path using `pkg-config`.
-On Debian-based systems like Ubuntu, this can usually be done via
-`sudo apt-get install pkg-config`.
+- macOS:
+
+  ```sh
+  $ brew install grpc
+  ```
+
+- Any OS: to build and install from sources, refer to [Building from
+  source][BUILDING].
 
 #### Protocol Buffers v3
 
-While not mandatory to use gRPC, gRPC applications usually leverage Protocol
-Buffers v3 for service definitions and data serialization, and our example code
-uses Protocol Buffers as well as gRPC. If you don't already have it installed on
-your system, you can install the version cloned alongside gRPC. First ensure
-that you are running these commands in the gRPC tree you just built in the from
-the previous step.
+While not mandatory, gRPC applications usually leverage [Protocol Buffers
+v3][pbv3] for service definitions and data serialization, and the example code
+uses Protocol Buffers.
+
+- macOS (if you installed `grpc` using brew, then you can skip this step since
+  `protobuf` is installed as a prerequisite):
+
+  ```sh
+  $ brew install protobuf
+  ```
+
+- Any OS:
+
+   1. Download a zip file of the latest version of pre-compiled binaries for
+      your operating system from [github.com/google/protobuf/releases][]
+      (`protoc-<version>-<os>.zip`).
+   2. Unzip the file.
+   3. Update your environment's path variable to include the path to the
+      `protoc` executable.
+
+If you've built gRPC from sources, you have another option. You can install
+protobuf from its submodule in the `third_party` folder:
 
 ```sh
 $ cd third_party/protobuf
 $ make && sudo make install
```

Comment:
I wouldn't recommend to install  protobuf library  at a system level (=i.e. without a prefix & using a local directory). There is no ways of uninstalling and it will forever pollute your system with a specific version of protobuf (which will cause a lot of problems once trying to upgrade protobuf)

Reply:
Ideally, one would install both gRPC and protobuf using one of the approaches mentioned in
https://github.com/grpc/grpc/tree/master/src/cpp#cmake

When installing, one should always use CMAKE_INSTALL_PREFIX to avoid a system-wide installation.

Reply:
Btw, with newer version of cmake, once can easily install both protobuf and grpc by a single command.
https://github.com/grpc/grpc/tree/master/src/cpp#fetchcontent

Reply:
> I wouldn't recommend to install protobuf library at a system level ...

Ok, I've removed these instructions for now.

> Ideally, one would install both gRPC and protobuf using one of the approaches mentioned in ... 
> ... [or using fetchcontent]

Agreed, but I'd like to propose that we address that in the next round of edits. WDYT @jtattermusch?

Reply:
Ok, I've updated the instructions to explain how to do a local install using `CMAKE_INSTALL_PREFIX`. PTAL